bedrock/exp/redirects.py

```diff
@@ -0,0 +1,7 @@
+from bedrock.redirects.util import redirect
+
+redirectpatterns = (
+
+    redirect(r'^exp/firefox/new/nav?$', 'firefox.new'),
```

Comment:
The ending `v` on `nav` is optional here, should this be a trailing slahs instead? Having a test for this redirect would be good too, but not a blocker.

```suggestion
    redirect(r'^exp/firefox/new/nav/?$', 'firefox.new'),
```